lib/mintacoin/accounts/accounts.ex

```diff
@@ -81,6 +64,20 @@ defmodule Mintacoin.Accounts do
     |> Repo.insert()
   end
 
+  @spec create_trustline(params :: params()) :: {:ok, asset_holder()} | {:error, error()}
+  def create_trustline(%{
+        asset: %{id: asset_id, code: code},
+        trustor_wallet: %Wallet{id: trustor_wallet_id} = trustor_wallet,
+        signature: signature
+      }) do
+    with {:ok, encrypted_secret_key} <- system_encrypt_secret_key(trustor_wallet, signature) do
+      trustor_wallet_id
+      |> AssetHolders.retrieve_by_wallet_id_and_asset_id(asset_id)
+      |> process_trustor(asset_id, trustor_wallet)
+      |> dispatch_create_trustline_job(encrypted_secret_key, code)
+    end
```

Comment:
@jhonatan-kmt let's rewrite this function, i suggest the following change:
```suggestion
    case system_encrypt_secret_key(trustor_wallet, signature) do
      {:ok, encrypted_secret_key} ->
        trustor_wallet_id
        |> AssetHolders.retrieve_by_wallet_id_and_asset_id(asset_id)
        |> process_trustor(asset_id, trustor_wallet)
        |> dispatch_create_trustline_job(encrypted_secret_key, code)
      {:error, error} -> {:error, error}
    end
```

This implementation completes 100% of the test coverage of the module.

Reply:
I like it, I will implement it. Thanks for the suggestion